src/components/item-details/bid-tab/delisting-tab-content.tsx

```diff
@@ -0,0 +1,93 @@
+import Button from '@ui/button';
+import { SpinnerContainer } from '@ui/spinner-container/spinner-container';
+import { useState } from 'react';
+import Modal from 'react-bootstrap/Modal';
+import useMetamask from '../../../features/auth/hooks/useMetamask';
+import { delistItem } from '../../../features/marketplace/store/marketplace.actions';
+import useAppDispatch from '../../../store/hooks/useAppDispatch';
+import useAppSelector from '../../../store/hooks/useAppSelector';
+import { Item } from '../../../types/item';
+
+type Props = {
+  item: Item;
+};
+
+export const DelistingTabContent = ({ item }: Props) => {
+  useMetamask();
+  const { isLoading } = useAppSelector((state) => state.marketplace);
+  const [show, setShow] = useState(false);
+  const dispatch = useAppDispatch();
+
+  const onSubmit = async () => {
+    dispatch(
```

Comment:
Maybe it is good idea to include the withProtection action in here, so we double check that the action should be performed by an authenticated user.